PerformanceTests/Sources/PerformanceTestRunner/PerformanceMain.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import ClientRuntime
+
+// MARK: - Main Entry Point
+@main
+struct PerformanceMain {
+    static func main() async throws {
+        let runner = PerformanceTestRunner(iterations: 5)
+        let commitId = ProcessRunner.getGitCommitId()
+        let sdkVersion = ProcessRunner.getSdkVersion()
+
+        // Add more tests here
+        let performanceTests = [
+            AWSSTSGetCallerIdentity()
+        ]
+
+        var results: [PerformanceTestResult] = []
+        for test in performanceTests {
+            let result = try await runner.runTest(test)
+            results.append(result)
+        }
```

Comment:
nit: Map list of tests directly into list of results.
```suggestion
        let results = performanceTests.map { try await runner.runTest($0) }
```

Reply:
oh actually I remember I tried that first but I get an error:
`Cannot pass function of type '(AWSSTSGetCallerIdentity) async throws -> PerformanceTestResult' to parameter expecting synchronous function type`